app/views/api_docs/pages/home.md

```diff
@@ -0,0 +1,163 @@
+---
+page_title: Register API documentation
+title: Register API documentation
+---
+
+This is API documentation for the Department for Education (DfE)’s Register
+trainee teachers service.
+
+Register has replaced the HESA service for teacher trainees. All vendors of
+student record systems (SRS) and some training providers will need to make
+changes to integrate with Register.
+
+# What this API is for
+
+The Register API allows providers to import trainee records from their student
+record systems and to keep those records synchronised as they are modified.
+
+Providers can use the API for:
+
+- Looking up a trainee record in Register and returning details about that trainee.
+- Retrieving a list of trainee records in Register for a given academic year.
+- Creating new trainee records.
+- Updating individual trainee records.
+- Withdrawing and deferring a trainee.
+- Looking up reference data such as country code, school URNs etc.
+
+# What the API doesn't support
```

Comment:
Do we need this section?

Reply:
If we end up getting a few re-occuring questions re: api use, we could add the answers here but I'd say probably not now.

Reply:
Will remove for now